File: 1.c
/* find the sum of all ints less than 1,000 that are
 * evenly divisible by 3 or 5
 */

#include <stdio.h>

#define LIMIT 1000

int main() {

    int sum = 0;

    for(int i = 0; i < LIMIT; i++) {

        if(i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }

    printf("%d\n", sum);
}

File: 1.rs
/* find the sum of all ints less than 1,000 that are
 * evenly divisible by 3 or 5
 */

fn main() {
    let mut sum: i32 = 0;

    for x in 0..1000 {
        if x % 3 == 0 || x % 5 == 0 {
            sum += x;
        }
    }

    println!("sum -> {}", sum);
}
